.github/workflows/release-desktop.yml

```diff
@@ -26,9 +26,10 @@ jobs:
           echo '${{ secrets.api_key }}' > ~/private_keys/AuthKey_${{ secrets.api_key_id }}.p8
           echo '${{ secrets.mac_certs }}' | base64 -d > applecert.p12
 
-      - name: Extract platform name
-        id: get-os
-        run: echo "os=$(echo ${{ matrix.os }} | cut -d- -f1)" >> $GITHUB_OUTPUT
+      - name: Extract args
+        if: startsWith(matrix.os, 'macos')
+        id: get-args
+        run: echo "args=--universal" >> $GITHUB_OUTPUT
```

Comment:
We want to pass the `--universal` flag to the [electron-builder command](https://github.com/nop33/action-electron-builder/blob/master/index.js#L139-L140) through the `args` argument of the `nop33/action-electron-builder` action and we only want it for the macOS build, not windows or linux.